charts/horizon/values.yaml

```diff
@@ -56,11 +56,24 @@ ingest:
   cliArgs:
   - "--apply-migrations"
 
-  ## This variable allows overrides of default settings
-  horizonConfig:
+  ## Horizon config
+  config:
```

Comment:
This change would have 2 consequences:
1. It would limit usability of this chart as subchart. This is something we wanted to use to build full stacks that include validators and friendbot
2. Such option rename could break people's workloads

For those reasons I think it would be best to collaborate on a plan for the chart improvements before making this change. Once we have a plan we could communicate it to users to limit disruptions.